modules/kubernetes/prometheus/charts/prometheus-extras/templates/monitor.yaml

```diff
@@ -312,3 +312,37 @@ spec:
     - path: /metrics
       port: metrics
 {{- end }}
+{{- if .Values.enabledMonitors.csiSecretsStorProviderAzure }}
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: csi-secrets-store-driver-azure
+  namespace: csi-secrets-store-provider-azure
+  labels:
+    xkf.xenit.io/monitoring: platform
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/instance: csi-secrets-store-driver-azure
+      app.kubernetes.io/name: csi-secrets-store-driver-azure
+  podMetricsEndpoints:
+    - path: /metrics
+      port: metrics
+{{- end }}
+{{- if .Values.enabledMonitors.csiSecretsStorProviderAws }}
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: csi-secrets-store-driver-aws
+  namespace: csi-secrets-store-provider-aws
+  labels:
+    xkf.xenit.io/monitoring: platform
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/instance: csi-secrets-store-driver-aws
+      app.kubernetes.io/name: csi-secrets-store-driver-aws
```

Comment:
Today I don't define these values in the ds:
https://github.com/XenitAB/terraform-modules/blob/main/modules/kubernetes/csi-secrets-store-provider-aws/charts/csi-secrets-store-provider-aws/templates/daemonset.yaml

Please update the helm chart and don't forget to update: https://github.com/XenitAB/terraform-modules/blob/main/modules/kubernetes/csi-secrets-store-provider-aws/charts/csi-secrets-store-provider-aws/Chart.yaml

Set the values to:  csi-secrets-store-provider-aws to be consistent with azure